src/js/load-package.ts

```diff
@@ -599,8 +626,6 @@ if (typeof API !== "undefined" && typeof Module !== "undefined") {
   loadedPackages = singletonPackageManager.loadedPackages;
 
   // TODO: Find a better way to register these functions
-  API.recursiveDependencies =
```

Comment:
@hoodmane I don't remember why we've exposed this API (https://github.com/pyodide/pyodide/pull/4555). Are you using it somewhere downstream?

Reply:
Doesn't look like it. I didn't explain what I was up to in #4555 and you didn't ask so I guess we'll never know...